app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,10 @@
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
+<<<<<<< HEAD
```

Comment:
This Firehose video explains it: http://www.thefirehoseproject.com/advanced-git-topics/5

Melissa did fix it in her unmerged PR, you can see how she did it here: https://github.com/pawn-stars/chess-app/pull/18/commits/6cab7e763d135e375b580bc12394bc920175c68c